app/uk/gov/hmrc/cataloguefrontend/healthindicators/HealthIndicatorsConnector.scala

```diff
@@ -87,12 +94,13 @@ object Rating {
   }
 }
 
-case class RepositoryRating(repositoryName: String, repositoryScore: Int, ratings: Seq[Rating])
+case class RepositoryRating(repositoryName: String, repositoryType: String, repositoryScore: Int, ratings: Seq[Rating])
```

Comment:
Try and avoid `String` where an enumerated type would be preferable.
A `RepoType` classification is used below.  Can that be used here?